controllers/highlight.js

```diff
@@ -48,5 +49,15 @@ export default {
       return res.json({ message: 'You have no highlights yet!' });
     }
     return res.json({ highlights });
+  },
+
+  async removeHighlight(req, res) {
+    const { params: { id }, decoded } = req;
```

Comment:
I think `slug` should be part of the params we get a highlight with here. Maybe more of slug than id just as we did in the case of comments.

Reply:
I think we could have the highlight `id` too, however, it will be good to have slug as well.

Reply:
Okay, I'll do so

Reply:
I've added `slug` to the params